src/database/resolvers.js

```diff
@@ -1,48 +1,71 @@
-			args.criteria.lastQueryDate = args.criteria.lastQueryDate || new Date('2001-01-01')
-			args.criteria.lastArticleId = args.criteria.lastArticleId || '000000000000000000000000'
-
-			const promises = categories.map(async category => {
-				const _articles = await Article.find({
-					category,
-					link: { $ne: null },
-					modifiedDate: { $gt: new Date(args.criteria.lastQueryDate) },
-					_id: { $gt: args.criteria.lastArticleId },
-				})
-					.lean()
-					.populate('source')
-					.sort({ _id: -1 })
-					.limit(20)
-
-				return [..._articles]
-			})
-
-			const articles = await Promise.all(promises)
-
-			const articleFlatterend = _.flatten(articles)
-			const sortedArticles = await getSortedArticle(articleFlatterend)
-
-			return sortedArticles
-		},
-
-		getTweets: async (parent, args, { Tweet }) => {
-			args.criteria = args.criteria || {}
-			args.criteria.lastQueryDate = args.criteria.lastQueryDate || new Date('2001-01-01')
-			args.criteria.lastTweetId = args.criteria.lastTweetId || '000000000000000000000000'
-
-			const tweets = await Tweet.find()
-				.populate('twitterHandle')
-				.sort({ publishedDate: -1 })
-				.limit(100)
-
-			return tweets
-		},
-	},
-}
+  Query: {
+    getArticles: async (parent, args, { Article }) => {
+      args.criteria = args.criteria || {};
+      args.criteria.lastQueryDate =
+        args.criteria.lastQueryDate || new Date("2001-01-01");
+      args.criteria.lastArticleId =
+        args.criteria.lastArticleId || "000000000000000000000000";
+
+      const promises = categories.map(async category => {
+        const _articles = await Article.find({
+          category,
+          link: { $ne: null },
+          modifiedDate: { $gt: new Date(args.criteria.lastQueryDate) },
+          _id: { $gt: args.criteria.lastArticleId }
+        })
+          .lean()
+          .populate("source")
+          .sort({ _id: -1 })
+          .limit(20);
+
+        return [..._articles];
+      });
+
+      // console.log('Printing articles before sorting', articles.map(a => a.title))
+
+      const articles = await Promise.all(promises);
+
+      const articleFlatterend = _.flatten(articles);
+      const sortedArticles = await getSortedArticle(articleFlatterend);
+
+      return sortedArticles;
+    },
+
+    getTweets: async (parent, args, { Tweet }) => {
+      args.criteria = args.criteria || {};
+      args.criteria.lastQueryDate =
+        args.criteria.lastQueryDate || new Date("2001-01-01");
+      args.criteria.lastTweetId =
+        args.criteria.lastTweetId || "000000000000000000000000";
+
+      const tweets = await Tweet.find()
+        .populate("twitterHandle")
+        .sort({ publishedDate: -1 })
+        .limit(100);
+
+      return tweets;
+    }
+  },
+  Mutation: {
+    storeFcmToken: async (parent, args) => {
+      const {
+        input: { fcmToken, countryCode }
+      } = args;
+      console.log("_______________fcmtoken_______________", fcmToken);
```

Comment:
@shrijan-siris let's remove console.logs from final code. Also device token is a secret information, we probably don't want in logs